plugins/outputs/elasticsearch/elasticsearch.go

```diff
@@ -23,23 +23,27 @@ import (
 
 type Elasticsearch struct {
 	URLs                []string `toml:"urls"`
-	IndexName           string
-	DefaultTagValue     string
-	TagKeys             []string
-	Username            string
-	Password            string
-	AuthBearerToken     string
-	EnableSniffer       bool
-	Timeout             config.Duration
-	HealthCheckInterval config.Duration
-	EnableGzip          bool
-	ManageTemplate      bool
-	TemplateName        string
-	OverwriteTemplate   bool
-	ForceDocumentID     bool `toml:"force_document_id"`
-	MajorReleaseNumber  int
-	FloatHandling       string          `toml:"float_handling"`
-	FloatReplacement    float64         `toml:"float_replacement_value"`
+	IndexName           string   `toml:"index_name"`
+	DefaultTagValue     string   `toml:"default_tag_value"`
+	tagKeys             []string
```

Comment:
Can  you maybe group the "user-option" fields and the internal fields here? I.e. move `tagKeys` and `majorReleaseNumber` at the end of the struct? That's what most other plugins look like...

Reply:
I couldn't decide before if they should be done that way, or alphabetically, or...  So I ended up leaving it how it was..  Now they are done alphabetically grouped by User Config, locals, then core structs...